lib/ExpandablePicker/ExpandablePickerAccessibilitySupport.js

```diff
@@ -0,0 +1,27 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name ExpandablePickerAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	initAccessibility: kind.inherit(function (sup) {
+		return function (props) {
+			sup.apply(this, arguments);
+			this.$.currentValue.observe('content', this.valueChanged, this);
+			this.valueChanged();
+		};
+	}),
+
+	/**
+	* @private
+	*/
+	valueChanged: function() {
```

Comment:
We should avoid adding methods in the form xxxChanged that aren't property change handlers for properties on this object. If this, or a subkind, added a value property (or really, just called `set('value', 0)`), this method would be automatically called unintentionally.